e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -535,6 +536,33 @@ describeEE("scenarios > embedding > full app", () => {
         ).to.be.true;
       });
     });
+
+    it("should allow downloading question results when logged in via Google SSO (metabase#39848)", () => {
+      const CSRF_TOKEN = "abcdefgh";
+      cy.intercept("GET", "/api/user/current", req => {
+        req.on("response", res => {
+          res.headers["X-Metabase-Anti-CSRF-Token"] = CSRF_TOKEN;
+        });
+      });
+      cy.intercept(
+        "POST",
+        "/api/dashboard/*/dashcard/*/card/*/query/csv?format_rows=true",
+      ).as("CsvDownload");
+      visitDashboardUrl({
+        url: `/dashboard/${ORDERS_DASHBOARD_ID}`,
+      });
+
+      getDashboardCard().realHover();
+      getDashboardCardMenu().click();
+      popover().findByText("Download results").click();
+      popover().findByText(".csv").click();
+
+      cy.wait("@CsvDownload").then(interception => {
+        expect(
+          interception.request.headers["x-metabase-anti-csrf-token"],
```

Comment:
Then I only assert that we pass the correct header. As downloaded files are tested in other tests.